fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -2397,8 +2398,9 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
                                         "or set parallel_fragment_exec_instance_num to a lower value in session variable");
                     } else {
                         ErrorReport.reportTimeoutException(ErrorCode.ERR_TIMEOUT, getExecType(), timeout,
-                                String.format("please increase the '%s' session variable and retry",
-                                        SessionVariable.INSERT_TIMEOUT));
+                                String.format("please increase the '%s' session variable or the '%s' property for " +
```

Comment:
Add into ErrorCode.java?

Reply:
`ERR_TIMEOUT` is already in `ErrorCode` and this is error message in `ERR_TIMEOUT`